src/Styleguide/Components/Tabs.tsx

```diff
@@ -60,11 +78,11 @@ export class Tabs extends React.Component<TabsProps, TabsState> {
   }
 
   render() {
-    const { children = [] } = this.props
+    const { children = [], justifyContent = "left" } = this.props
```

Comment:
Instead of giving justifyContent a default here, how about we add it as a part of the class. This way if we ever integrate a tool to read in the default props it'll be picked up. 

```typescript
export class Tabs extends React.Component<TabsProps, TabsState> {
 public static defaultProps: Partial<TabsProps> = {
        justifyContent: "left"
    }
}
```

Reply:
Yes, that's better! Fixed in ee239365589aab88ef5c4837ba308cc12b2f1d88

Good to know about `Partial` too — thanks for the tip 😃